setup.cfg

```diff
@@ -1,3 +1,70 @@
+[metadata]
+name = falcon
+version = attr: falcon.__version__
+description = An unladen web framework for building APIs and app backends.
+long_description_content_type = text/x-rst
+url = https://falconframework.org
+author = Kurt Griffiths
+author_email = mail@kgriffs.com
+license = Apache 2.0
+license_file = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Web Environment
+    Natural Language :: English
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: Apache Software License
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Topic :: Internet :: WWW/HTTP :: WSGI
+    Topic :: Software Development :: Libraries :: Application Frameworks
+    Programming Language :: Python
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
This is missing classifiers for 3.9 and Cython. I didn't notice any other differences vs. the latest `setup()` kwargs, but it would be a good idea to double-check.

Reply:
do you have a classifier for cython?